ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -346,7 +346,7 @@ export class NodeDetailsContent extends React.Component {
             showValidation={showValidation}
             variableTypes={variableTypes}
             errors={fieldErrors}
-            inferredVariableType={inferredType && ProcessUtils.humanReadableType(inferredType)}
+            inferredVariableType={this.props.expressionType && ProcessUtils.humanReadableType(this.props.expressionType)}
```

Comment:
can we have null check inside humanReadableType?

Reply:
we can, but I'm not sure if it would be better that way.
but, yes, maybe that would be more JS idiomatic :)